output/mistral_influxdb/mistral_influxdb.c

```diff
@@ -93,9 +93,18 @@ static void usage(const char *name)
      */
     mistral_err("Usage:\n");
     mistral_err(
-        "  %s [-d database] [-h host] [-P port] [-e file] [-m octal-mode] [-u user] [-p password] [-s] [-v var-name ...]\n",
-        name);
+        "  %s [-d database] [-h host] [-P port] [-e file] [-m octal-mode] [-u user] [-p password] [-s] [-v var-name ...]\n"
```

Comment:
This summary omits the -k (skip SSL). Speaking of which, do we have that in the other plugins?